YouVsVirus/Assets/Materials/Physics/HumanColliderMat.physicsMaterial2D

```diff
@@ -7,5 +7,5 @@ PhysicsMaterial2D:
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_Name: HumanColliderMat
-  friction: 0.5
-  bounciness: 0.5
+  friction: 0.82
+  bounciness: 0.82
```

Comment:
This always changes due to social distancing. Not for now, but should we ignore it in git or is that too dangerous? Or maybe always reset it to the same value at the end of the game?

Reply:
ahhhhhhhhhhhhhhhhh
This finally explain why this file is always showing up in my git diff and it annoys me sooooo much.
So this is actually unintentionally committed here.

Reply:
does not matter, leave it, it does no harm